### ISSUE TITLE
Summarize Coral Observations to planning unit grids

### ISSUE THREAD

**Issue by rhodges** [Ecotrust/PEW-EFH#34]
- [x] add to PUG or back-door intersect query
- [ ] keep all 3 (1 still coming) layers separate
- [ ] be sure all three values are reported after drawing a shape

**Comment by rhodges** [comment]
Hi Jon and Ryan,

After further review, both of our coral datasets had no value for the “count” field for the vast majority of the points.  I took the subsets for both datasets that had a counted number of coral or sponge and recreated the Tilemill projects for each.  I have also summarized them both to the planning unit grid with two new columns showing the summed value of the counts.  The two columns are “cs_obs” (the larger point dataset) and “cs_b3500m” (corals/sponges below 3500m). 

Ryan, there are two new .xml files (one for each coral observation dataset) here:
G:\projects\PewEFH2015\Deliverables\Data\Mapnik_xml
The updated spatial data is here:
                G:\projects\PewEFH2015\Data\Processed\PEW_final_datasets\Corals_and_sponges (NOAA……..w_counts.shp)
The legends are still the same.

I will investigate the area calculations when I return and try to figure out the best solution.

Thanks,
Nick
